core/tauri/src/app.rs

```diff
@@ -1701,6 +1705,11 @@ fn on_event_loop_event<R: Runtime, F: FnMut(&AppHandle<R>, RunEvent) + 'static>(
     RuntimeRunEvent::Resumed => RunEvent::Resumed,
     RuntimeRunEvent::MainEventsCleared => RunEvent::MainEventsCleared,
     RuntimeRunEvent::UserEvent(t) => t.into(),
+    #[cfg(target_os = "macos")]
+    RuntimeRunEvent::OpenURLs(urls) => {
+      app_handle.trigger_global("open-urls", Some(to_string(&urls).unwrap()));
+      RunEvent::OpenURLs(urls)
```

Comment:
Compiler throws error because RuntimeRunEvent argument is Vec<string> and RunEvent argument is Vec<url::Url>